addons/tts-captions/addon.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Text to Speech Captions",
+  "description": "Adds live captions for the text to speech voice.",
+  "credits": [
+    {
+      "name": "GrahamSH"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "style.css",
+      "matches": ["projects"]
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["projects"],
+      "runAtComplete": false
+    }
+  ],
+  "dynamicEnable": false,
+  "dynamicDisable": false,
+  "versionAdded": "1.30.0",
+  "tags": ["projectPage"],
```

This should have the community tag, as mentioned in https://github.com/ScratchAddons/ScratchAddons/pull/5514#issuecomment-1367882657.

```suggestion
  "tags": ["community", "projectPage"],
```